app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,80 @@
+class OrdersController < ApplicationController
+  def show
+    id = params[:id]
+    @order = Order.find(id)
+    if @current_user != nil
```

Comment:
If `@current_user` isn't being used in the corresponding view, it shouldn't be an instance variable